packages/playwright/src/index.ts

```diff
@@ -147,7 +147,14 @@ export default class AxeBuilder {
       context
     );
     const partials = await partialResults.getPartials();
-    return this.finishRun(partials);
+
+    try {
+      return await this.finishRun(partials);
+    } catch (error) {
+      throw new Error(
+        `Error: ${error}\n Please check out https://github.com/dequelabs/axe-core-npm/blob/develop/packages/playwright/error-handling.md`
```

Comment:
I do not recall what the spec says here. Does `Error#toString()` _always_ return `error.message`? If not, we might want to be explicit here:

```suggestion
        `Error: ${error.message}\n Please check out https://github.com/dequelabs/axe-core-npm/blob/develop/packages/playwright/error-handling.md`
```